addons/spreadsheet/static/src/global_filters/plugins/global_filters_ui_plugin.js

```diff
@@ -404,7 +405,7 @@ export default class GlobalFiltersUIPlugin extends spreadsheet.UIPlugin {
     _getRelationDomain(filter, fieldMatching) {
         const values = this.getGlobalFilterValue(filter.id);
         if (!values || values.length === 0 || !fieldMatching.chain) {
-            return undefined;
+            return new Domain();
```

Comment:
docstring can be updated :)

Reply:
I'm wroried that ts does not detect that ...